### ISSUE TITLE
openvm-tools does not correctly identify CentOS7

### ISSUE THREAD

**Issue by jeroenjacobs1205** [vmware/open-vm-tools#311]
Hi,

I have two vm's running Centos7. One of those is correctly recognized in esx, bu the other shows me an error:

```
he configured guest OS (CentOS 7 (64-bit)) for this virtual machine does not match the guest that is currently running (CentOS 6 (64-bit)). You should specify the correct guest OS to allow for guest-specific optimizations.
```

The version of open-vm-tools on the vm that is identified correctly is: 10.1.10-3.el7_5.1
The version of open-vm-tools on the vm that is NOT identified correctly is: 10.2.5-3.el7

**Comment by oliverkurth** [comment]
Thank you for reporting. We recently discovered this issue in our testing too and are working on a fix.

There is a quick and dirty workaround - in the file `/etc/centos-release`, replace the version `7.6.1810` with `7.6`.

**Comment by jothoma1** [comment]
Same here, workaround work.
It breaks our CMDB provisionning :(

**Comment by evreneter** [comment]
> It breaks our CMDB provisionning :(

Hello,
What breaks your system jothoma1?  Is it safe to change content of this file?
Thanks.

**Comment by johnwvmw** [comment]
@jothoma1 

Would you please provide some details about what aspect of your CMDB provisioning breaks with either the incorrect  identification of CentOS 7.6 as a CentOS 6 release or the 'dirty' work-around. 
Thanks

**Comment by oliverkurth** [comment]
Closing - fix will be in the next release. If this is needed before that, you need to rebuild open-vm-tools with the fix (see above) applied.

**Comment by rodrigc** [comment]
@oliverkurth by any chance do you know when your fix will make it to an RPM available in the Centos repository?  I'm doing a `yum install open-vm-tools` on Centos 7.6 and 
am getting this version:

```
open-vm-tools-10.2.5-3.el7.x86_64
```

**Comment by oliverkurth** [comment]
@rodrigc see https://bugzilla.redhat.com/show_bug.cgi?id=1672087 . Looks like there won't be a fix in CentOS 7.6, but 7.7 does not have that problem.

**Comment by FlorianLeo** [comment]
> @rodrigc see https://bugzilla.redhat.com/show_bug.cgi?id=1672087 . Looks like there won't be a fix in CentOS 7.6, but 7.7 does not have that problem.

Oh yes it does - at least mine does
